lib/internal/child_process.js

```diff
@@ -409,7 +409,14 @@ ChildProcess.prototype.unref = function() {
 
 
 function setupChannel(target, channel) {
-  target._channel = channel;
+  target.channel = channel;
+
+  Object.defineProperty(target, '_channel', {
+    get() { return target.channel; },
+    set(val) { target.channel = val; },
```

Comment:
Setting it is DEEP into undefined behaviour, I guess its backwards compatible... but I can't think of a reason to take a shot-gun to the ChildProcess internals.